lua/harpoon/ui.lua

```diff
@@ -81,15 +91,30 @@ function M.toggle_quick_menu()
     local contents = {}
     local global_config = harpoon.get_global_settings()
 
+    local formatter_method = global_config.formatter
+        and global_config.formatter.method
+    local formatter = formatters[formatter_method]
+        and formatters[formatter_method](
+            global_config.formatter and global_config.formatter.payload or {}
+        )
+
     Harpoon_win_id = win_info.win_id
     Harpoon_bufh = win_info.bufnr
 
     for idx = 1, Marked.get_length() do
         local file = Marked.get_marked_file_name(idx)
-        if file == "" then
-            file = "(empty)"
+        local path = string.format("%s", file)
+        local label = formatter and formatter(file) or file
```

Comment:
thet thing that is tripping me up is why would there not be a formatter?

Can we just make a method that always ensure there is a formatter?

Reply:
that is all my feedback.  I just think that we already have an identity function for the default.  Make it so then.